.lintstagedrc.json

```diff
@@ -1,6 +1,7 @@
 {
-  "*.{js,jsx,ts,tsx,json,scss}": [
+  "*.{js,jsx,ts,tsx,json}": [
     "eslint src --ext .ts,.tsx",
-    "prettier --check \"src/**/*.{js,jsx,ts,tsx,json,scss}\""
-  ]
+    "prettier --check \"src/**/*.{js,jsx,ts,tsx,json}\""
+  ],
+  "*.{css,scss}": ["prettier --check \"src/**/*.{css,scss}\""]
```

Comment:
prettier handles linting and formatting of scss fine, but i saw mentions in most guides talking about [stylelint](https://stylelint.io/user-guide/cli). Is this something that'd be worthwhile adding if prettier is already taking care of linting + formatting?

Reply:
I think I'd consider it out of scope for this PR, but happy to accept a subsequent PR that adds stylelint following their recommended standards for SCSS

Reply:
Prettier and stylelint check different things, so it might be interesting. I tried stylelint right now. There are a few syntax changes for example using the newer space separated color functions `rgba(0 0 255 / 80%)` instead of `rgba(0, 0, 255, 0.8)` etc. that i would call "nice-to-have". it also warns a lot about using CamelCase for identifiers instead of kebab-case. That would need to be refactored or configured. Imho the amount of scss in this repo is so small we dont really need a linter, but it would not hurt.